### ISSUE TITLE
How to Disable Title Case Transformation on Double-Click in VSCode and Request for Mouse Event Support in Keyboard Shortcuts

### ISSUE THREAD

**Issue by bryantyan** [getcursor/cursor#529]
When I double-click to highlight text, the text transforms to title case. However, this does not happen in VSCode. How can I disable this feature? 

Also, keyboard shortcuts cannot define or query mouse events. Can this feature be considered?

**Comment by lixinweijy** [comment]
why not free it?

**Comment by danperks** [comment]
I don't have this issue, so will assume this is unique to you for now. Besides a fresh install and resetting your settings, not sure what to suggest here. Maybe a shortcut in your mouse software (if you have any?).

**Comment by bryantyan** [comment]
Problem solved, thanks to the advice from @danperks . I reinstalled and reset the settings.

- The surface issue was caused by installing the awesome emacs extension and activating one of its settings.
- The actual issue was that the mouse double-click event for the cursor was bound to the **cmd-c** shortcut command. Therefore, when the extension modified the command bound to **cmd-c**, emacs-mcx.transformToTitlecase, it caused the text to become title case when double-clicked with the mouse.
- VSCode has the same design, and I was able to reproduce the issue.

Conclusion: If you want to change the mouse double-click event, just bind cmd-c to the command you prefer.

Very interesting.
